### ISSUE TITLE
Add OS integration for and revamp quickload abilities

### ISSUE THREAD

**Issue by techninja** [evil-mad/robopaint#120]
Home user directories are somewhat standardized across OS' and can be used to create/reference a directory used for quickload. This would require some additional changes to how quickload works, like scrolling, folder and filename management, but I think it would be a boon to making working with files far simpler.

Basic feature description:
- By clicking Quickload, a user is presented with a hover/click sensitive hierarchical menu of folders:
  - **Examples**: Default embedded images (could also be loaded into the "root", but that clutters things up a bit)
  - **User**: Pointing to `%USERHOME%/Robopaint/svgs`, possibly created on install or on prompt. Lists all *.svg files in folder. Possibly flattening all subfolder content as well, or allowing for actual hierarchy.
  - **Interweb**: Out of left field, would display stuff tagged "watercolorbot" from thingiverse using their API, or perhaps pull from some other public contributable resource.
- Another Plus: could provide some small preview of the svg. Seeing as these could just as well be extremely small `<img>` tags directly referencing the svgs themselves, this should be a surprisingly easy and useful add.

**Comment by docprofsky** [comment]
Is the examples Quickload different from the current examples menu?

**Comment by techninja** [comment]
Yep. It used to just be the text list, but now it's the thumbnail picture list. This ticket is mostly about making that same feature as useful as possible. Add some default folder listing, with slide out side menus. a bit like a native quick load "recent files" menu, though likely more visual. Could easily use some modern JS library I can't seem to find. This is actually low level enough and the APIs are clean enough that improvement of this feature in RP could actually be done by someone somewhat unfamiliar with the project!

**Comment by techninja** [comment]
Not the getting more examples thing though, that would still need to be done by someone who's done lots of prints on the various bots and has good CC or public domain art available
